### PR TITLE
Remove Embedded Native Dlls from managed Scintilla dll

### PR DESCRIPTION
**Pull request by RobSwDev**
Remove Embedded Native Dlls from managed Scintilla dll
They are copied as part of the build or nuget restore process now

I branched the original repo at 73fd0cfdebc8518cd0fff33e59138ac453d41bef (tag: v.5.3.2.1) 
(there are conflicts against master due to the changes since the introduction of the new abstractions assembly)

### CODE REVIEW

File: native/x64/version.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Scintilla: 5.1.5
+Lexilla: 5.1.4
```
Comment (VPKSoft) [inline native/x64/version.txt:2]: This also has different version:
image

File: native/x86/version.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Scintilla: 5.1.5
```
Comment (VPKSoft) [inline native/x86/version.txt:1]: The version conflicts with the DLL files as above.

File: native/x86/version.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
+Lexilla: 5.1.4
```
Comment (VPKSoft) [inline native/x86/version.txt:2]: The version conflicts with the DLL files as above.

**Review by VPKSoft — APPROVED** [review]

**Review by VPKSoft — CHANGES_REQUESTED** [review]
Something seems to be wrong with the DLL versions. Otherwise looks good.

File: native/x64/version.txt [hunk 4]
```diff
@@ -0,0 +1,2 @@
+Scintilla: 5.1.5
```
[the remark this answers is not in the document]
Comment (RobSwDev) [reply native/x64/version.txt:1]: These are simply the existing files, copied from elsewhere in the repository.
I.e. from `Shared\x64\Scintilla.zip` and `Shared\x64\version.txt`
I didn't change anything. Were the versions in the text file already out of sync with the dll properties?

**Review by VPKSoft — APPROVED** [review]

**Comment by VPKSoft** [comment]
Hi,
I should have tested this better, the WinForms designer seems to have stopped working with the 5.3.2.2 version. I'm going to have to roll back.

**Comment by VPKSoft** [comment]
See:
image

**Comment by VPKSoft** [comment]
Sorry about that, I can't let the designer to be broken so the changes from the release v.5.3.2.2 are now rolled back.

**Comment by RobSwDev** [comment]
Hi @VPKSoft 
Apologies - I don't use the designer, and so didn't check for that.
The reason it fails is that the path to the native libraries is not correct in the designer: the designer uses the dll in the nuget package cache rather than the build directory.
I can think of a couple of possible fixes:
1. Add better logic to locate the native dlls
2. Duplicate the native dlls in the nuget package so that they are always available in the expected relative location

I can implement 1 and resubmit a PR?

**Comment by VPKSoft** [comment]
Thank you - I checked out the new PR and it works fine now 👍

**Pull request merged**